nearcore/src/config.rs

```diff
@@ -1058,6 +1060,20 @@ pub fn init_configs(
                 CryptoHash::default(),
             );
             add_protocol_account(&mut records);
+            let shards = if num_shards > 1 {
+                ShardLayout::v1(
+                    (0..num_shards - 1)
+                        .map(|f| {
+                            AccountId::from_str(format!("shard{}.test.near", f).as_str()).unwrap()
```

Comment:
nit: maybe we should generate these account dividers randomly and sort them. In the current way, the middle shards will only have one account.

Reply:
good point

Reply:
these are not 'dividers' but 'fixed_shards' - and they have one account on purpose (as this makes later load-testing a lot easier).

So in the current setup, if we ask for 3 shards, we get:
shard 0: only account named 'shard0.test.near'
shard 1: only account named 'shard1.test.near'
shard 2: all the other accounts.